### PR TITLE
Add Room Support Documentation

### PR DESCRIPTION
**Pull request by funkybunch**
## Proposed change

Adding documentation for a new room targeting service in the SharkIQ integration.
Also, some general housekeeping & cleanup. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/89350
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
Please split the cleanup from adding new documentation into separate PRs. Those are not directly related.

../Frenck

**Comment by funkybunch** [comment]
Moved cleanup content into https://github.com/home-assistant/home-assistant.io/pull/26548

File: source/_integrations/sharkiq.markdown [hunk 1]
```diff
@@ -28,14 +28,36 @@ This platform has been tested and is confirmed to be working with the Shark IQ R
...
+    - "Entry"
+    - "Living Room"
+target:
+  entity_id: vacuum.my_vacuum
+```
+**Important Note:** The list of rooms must be written _exactly_ as they appear in the Shark Clean app. If you want exact names that the service will understand,
+look for the `Rooms` attribute of your Shark Robot Vacuum in Home Assistant.  This will be populated with the rooms that the robot has configured, as they will be
+understood by the service.
+
+If you want to use the area selector in service's the UI, you need to format the area's name exactly as they appear in the vacuum's `Rooms` attribute.  You may also check the attribute using Home Assistant's [Developer Tools](https://www.home-assistant.io/docs/tools/dev-tools/).
```
Comment (c0ffeeca7) [inline source/_integrations/sharkiq.markdown]:
```suggestion
If you want to use the area selector in the service's UI, you need to format the area names exactly as they appear in the vacuum's `Rooms` attribute.  You may also check the attribute using Home Assistant's [Developer Tools](https://www.home-assistant.io/docs/tools/dev-tools/).
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @funkybunch :+1:

**Comment by c0ffeeca7** [comment]
A merge conflict emerged. Please resolve the conflict.

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**